tools/funcslower_example.txt

```diff
@@ -75,9 +75,38 @@ This shows the first two arguments to __kmalloc -- the first one is the size
 of the requested allocation. The return value is also shown (null return values
 would indicate a failure).
 
+# ./funcslower.py -U -u 100 'c:open'
+Tracing function calls slower than 100 us... Ctrl+C to quit.
+COMM           PID    LAT(us)             RVAL FUNC
+ranlib         23857   910.01                6 c:open
+    b'llvm::sys::fs::openFileForRead(llvm::Twine const&, int&, llvm::SmallVectorImpl<char>*)'
+    b'[unknown]'
+    b'[unknown]'
+ar             24259  17460.29                8 c:open
```

Comment:
All these examples have `[unknown]`. Could you produce some examples without `[unknown]`? Thanks!

Reply:
I updated the example to show a more real use case. Originally I went with llvm and c:open because I wanted an example everyone could reproduce, but unfortunately most projects are not compiled with framepointers : /